### PR TITLE
Added default timeout for lambdas

### PR DESCRIPTION
**Pull request by ACarsonBull**
This PR makes it so when you create and run a lambda that it uses the default timeout(currently 3 second) unless otherwise specified. It relates to [this](https://github.com/localstack/localstack/issues/584) issue.

Tests: I didn't add tests because it is just adding a default value when finding an object in the data dictionary.

### CODE REVIEW

File: localstack/services/awslambda/lambda_api.py [hunk 1]
```diff
@@ -49,7 +49,7 @@
 LAMBDA_DEFAULT_HANDLER = 'handler.handler'
 LAMBDA_DEFAULT_RUNTIME = LAMBDA_RUNTIME_PYTHON27
 LAMBDA_DEFAULT_STARTING_POSITION = 'LATEST'
-LAMBDA_DEFAULT_TIMEOUT = 60
+LAMBDA_DEFAULT_TIMEOUT = 3000
```
Comment (whummer) [inline localstack/services/awslambda/lambda_api.py]: Looks like this value should be in seconds, rather than milliseconds?

**Comment by whummer** [comment]
Thanks for this PR @ACarsonBull ! Can you please take a look at the comment, regarding timeout in seconds? thanks

**Comment by whummer** [comment]
Trying one more time - @ACarsonBull any chance you can look into the issue mentioned above?

File: localstack/services/awslambda/lambda_api.py (hunk 1 above)
Comment (a-zharkov) [reply localstack/services/awslambda/lambda_api.py]: Confirm. The default timeout should be in seconds (checked via awscli, serverless and cloud formation).

**Comment by whummer** [comment]
Superseded by #1254 - we'll preserve the original commit in this PR.

**Pull request closed without merging**